osquery/tables/applications/posix/docker.cpp

```diff
@@ -409,8 +409,11 @@ QueryData genContainers(QueryContext& context) {
           BIGINT(container_details.get_child("State").get<pid_t>("Pid", -1));
       r["started_at"] = container_details.get_child("State").get<std::string>(
           "StartedAt", "");
-      r["finished_at"] = container_details.get_child("State").get<std::string>(
-          "FinishedAt", "");
+      if (r["state"] != "running") {
+        r["finished_at"] =
+            container_details.get_child("State").get<std::string>("FinishedAt",
+                                                                  "");
+      }
```

Comment:
I want to note that this means running containers will have `NULL`, instead of `""` for `finished_at`. I always think this makes sense, though the ToB always tell me there's some internal path that doesn't like NULLs. And yet, we have them all over.

Reply:
@directionless are you cool with that? Or should we change it to empty string? Just hoping to get this PR moving again.

Reply:
Would it make more sense to set this to an empty string? I do see nulls in other tables but it wouldn't be much more complicated to set it to an empty string.

Reply:
I'd hope one of @alessandrogario or @Smjert would have piped up. 

Anyhow i approved. I like null better, but we're just not consistent about it.